addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1635,6 +1635,8 @@ var KanbanFieldMany2ManyTags = FieldMany2ManyTags.extend({
         _.each(this.value.data, function (m2m) {
             var $tag = $('<span>')
                     .attr('title', _.str.escapeHTML(m2m.data.display_name))
+                    .text( _.str.escapeHTML(m2m.data.display_name))
```

Comment:
Is title still needed if this is now shown as visible text ? (I hate a text that says "blabla" and that open a popup that says "blabla" on hover).